app/controllers/hunters_controller.rb

```diff
@@ -32,7 +33,8 @@ def new
   # GET /hunters/1/edit
   #
   # @param id [Integer] id of the Hunter to be edited
-  def edit() end
+  # Available_gear contains all gear available to the Hunter's current playbook
```

Comment:
Comment leftover.

Reply:
Fixed with commit d7d6308.

Reply:
Same here, I'm still seeing this in the final file changes.

Reply:
Should be fixed as of 59ce133.